core/src/execution/datafusion/spark_hash.rs

```diff
@@ -104,66 +108,110 @@ fn test_murmur3() {
     let _expected = vec![
         142593372, 1485273170, -97053317, 1322437556, -396302900, 814637928,
     ];
+    assert_eq!(_hashes, _expected)
```

Comment:
We can remove `_` from `_hashes` and `_expected`. I am not sure why we don't compare them before, seems to disable the test so they are prefixed with `_`.